src/components/Table/components/TableHead/tableHead.tsx

```diff
@@ -28,27 +29,38 @@ export const TableHead = <T extends RowData>({
             <MTableRow>
               {headerGroup.headers.map(({ column, getContext, id }) => {
                 const { columnDef, getIsGrouped, getIsSorted } = column;
+                const annotation =
```

Comment:
@MillenniumFalconMechanic we should be able to get `annotation` from `columnDef.meta` on line 31.

Reply:
Niiiice